docs/releases/1.9.6.txt

```diff
@@ -21,3 +21,6 @@ Bugfixes
 
 * Fixed a regression where ``SessionBase.pop()`` returned ``None`` rather than
   raising a ``KeyError`` for nonexistent values (:ticket:`26520`).
+
+* Fixed a regression with the cached template loader when dealing with
```

Comment:
perhaps a bit simpler and more specific: "with" -> "causing" & "when dealing with" -> "to crash when using"